node_app/models/Organization.js

```diff
@@ -0,0 +1,36 @@
+/*
+Organization {
+  active : boolean
+  type : enum (prov, bus, govt, edu, reli, cg, other)
+  name : string
+}
+{
+  "Organization":
+    {
+      "active" : true,
+      "type" : "cg",
```

Comment:
I'm not sure if this would be cg or govt...?  I assume that a county health department would be govt?

Reply:
Lins 7-14 is just an example. The Organization.type valueset is from https://www.hl7.org/fhir/valueset-organization-type.html and cg stands for community group.